### DISCUSSION TITLE
Recognition of multiple languages in the same sentence

### DISCUSSION THREAD

**Discussion by matteoPiccoli** [PaddlePaddle/PaddleOCR#7612] [Q&A]
Hello everyone! 
I was wondering if by any chance it is possible to *recognize* two languages in the same sentence. I tried with korean+english and russian+english, but the result was not satisfactory as it happens for chinese+english model. 

I mean text like this in the image:

- 연비 (km/kWh)
- Аудиоустройство Bluetooth не подключено

I also tried using PULC in PaddleClas, but korean was recognized as japanese so it is not that useful. 

Anyone can help me?

**Comment by tink2123** [comment]
Thanks for your attention, the language classification model is being optimized, and this problem will be fixed in the next version.

**Reply by bely66** [reply]
Hi @tink2123 
So I'm assuming when recognizing text, instead of having a one recognition model that recognizes both language characters.
You're using a language classification model and based on the language you spin up the suitable model.

**Reply by PhenomenaPh** [reply]
Hi! @tink2123
Any progress on this issue? Also dealing with multiple languages in one photo. How can i pick both languages or specify only one which will return only text with specified language

**Reply by jayv0509** [reply]
I found out that you use dbnet which detects the text in a bigger region like complete sentences for example "hello, world", but if we use CRAFT instead which is more likely to detect them as a single world like "hello," and "world", two different worlds , can it fix the issue?
